core/src/main/scala/org/apache/spark/sql/execution/columnar/JDBCAppendableRelation.scala

```diff
@@ -105,13 +104,13 @@ case class JDBCAppendableRelation(
 
   // TODO: Suranjan currently doesn't apply any filters.
   // will see that later.
-  override def buildScan(requiredColumns: Array[String],
-      filters: Array[Filter]): RDD[Row] = {
+  override def buildUnsafeScan(requiredColumns: Array[String],
+                         filters: Array[Filter]): RDD[InternalRow] = {
```

Comment:
Formatting not proper (previous one was correct). Probably not using SnappyData's codeStyleSettings.xml, so correct as mentioned in build-instructions.md